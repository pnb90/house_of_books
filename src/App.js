import React from 'react'
import './App.css'
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
import styled from 'styled-components'
import Home from './views/Home/index'

const Background = styled.div`
  background-image: linear-gradient(to bottom, #aed581, #b2d788, #b7da8f, #bbdc95, #bfde9c, #bfde9c, #bfde9c, #bfde9c, #bbdc95, #b7da8f, #b2d788, #aed581);  min-height: 100%;
  min-width: 100%;
  background-size: cover;
  position: fixed;
`

function App() {
  return (
    <Router>
      <Background>
        <Switch>
          <Route exact path="/" component={ Home } />
        </Switch>
      </Background>
    </Router>
  );
}

export default App;
