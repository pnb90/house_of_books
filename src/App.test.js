import React from 'react';
import App from './App';
import { render } from '@testing-library/react'


it('renders without crashing', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot()
});
