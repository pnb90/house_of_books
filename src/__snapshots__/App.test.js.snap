// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    class="sc-cSHVUG fiViev"
  >
    <div
      class="sc-fjdhpX bDUtko"
    >
      <div
        class="sc-jzJRlG ivLtnB"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 11.55C9.64 9.35 6.48 8 3 8v11c3.48 0 6.64 1.35 9 3.55 2.36-2.19 5.52-3.55 9-3.55V8c-3.48 0-6.64 1.35-9 3.55zM12 8c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3z"
          />
        </svg>
        House of Books
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 11.55C9.64 9.35 6.48 8 3 8v11c3.48 0 6.64 1.35 9 3.55 2.36-2.19 5.52-3.55 9-3.55V8c-3.48 0-6.64 1.35-9 3.55zM12 8c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3z"
          />
        </svg>
      </div>
      <form
        data-testid="searchbar testID"
      >
        <label
          class="sc-bwzfXH bGgvvP"
          for="searchInput"
        >
           Book Search: 
        </label>
        <div
          aria-label="book searchbar"
          class="MuiInputBase-root sc-bdVaJa fcBMVU"
        >
          <input
            class="MuiInputBase-input"
            id="searchInput"
            placeholder="Search for books"
            type="text"
            value=""
          />
        </div>
      </form>
      <div
        class="sc-jTzLTM kwYROD"
        data-testid="displayContainer testID"
      />
    </div>
  </div>
</div>
`;
